Chap_API_Server.tex

```diff
@@ -963,10 +965,15 @@ \subsection{\code{PMIx_server_deregister_resources}}
 
 Remove information about resources not associated with a given namespace from the \ac{PMIx} server library. Only the \refarg{key} fields of the provided \refarg{info} array shall be used for the operation - the associated values shall be ignored except where they serve as qualifiers to the request. For example, to remove a specific fabric device from a given node, the \refarg{info} array might include a \refattr{PMIX_NODE_INFO_ARRAY} containing the \refattr{PMIX_NODEID} or \refattr{PMIX_HOSTNAME} identifying the node hosting the device, and the \refattr{PMIX_FABRIC_DEVICE_NAME} specifying the device to be removed. Alternatively, the device could be removed using only the \refattr{PMIX_DEVICE_ID} as this is unique across the overall system.
 
+\returnsimple
+
 \advicermstart
 As information not related to namespaces is considered \emph{static}, there is no requirement that the host environment deregister resources prior to finalizing the \ac{PMIx} server library. The server library shall properly cleanup as part of its normal finalize operations. Deregistration of resources is only required, therefore, when the host environment determines that client processes should no longer have access to that information.
 
 A \code{NULL} \refarg{cbfunc} reference indicates that the function is to be executed as a blocking operation.
+% TODO: If this is one of those functions that can be blocking or non-blocking, then
+% it needs a full return code explanation, not the returnsimple 
+
```

Comment:
Note during July 20 Meeting
 * Leave this todo as a reminder for a second effort to add more detail here.
 * Adding detail is not the goal of this PR, just cleanup/streamlining. So it was out of scope for this effort.